evaluate2.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+#
+#  Copyright 2023 The original authors
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+if [ -z "$1" ]
+  then
+    echo "Usage: evaluate2.sh <fork name> (<fork name 2> ...)"
+    echo " for each fork, there must be a 'prepare_<fork name>.sh' script and a 'calculate_average_<fork name>.sh' script"
+    echo " there may be an 'additional_build_steps_<fork name>.sh' script too"
+    exit 1
+fi
+
+function check_command_installed {
+  if ! [ -x "$(command -v $1)" ]; then
+    echo "Error: $1 is not installed." >&2
+    exit 1
+  fi
+}
+
+check_command_installed java
+check_command_installed mvn
+check_command_installed hyperfine
+check_command_installed jq
+
+set -o xtrace
+
+java --version
+
+mvn --quiet clean verify
```

Comment:
Maybe move this into fork's build.sh - thise way benchmark would support other platforms.

Reply:
E.g. build.sh may use Docker to build binary into fork/bin and then run.sh would be benchmarked.

Reply:
That would mean though we'd do the `mvn` build for each contender when running multiple (or even all) of them at once. I'd prefer if we didn't do that, so as to keep overall runtime low. In regards to Docker builds, it's a tempting idea. Can we keep it though for a separate time, I'm a bit concerned about scope creep and would rather not add further axes of freedom to the challenge at this point :)

Reply:
FWIW there are >20 non-java submissions and I think it would be interesting to see how they stand along.